build.rs

```diff
@@ -18,27 +18,41 @@ mod generate {
     use regex::Regex;
     use std::env;
     use std::fs::File;
-    use std::path::Path;
+    use std::path::{Path, PathBuf};
 
     static CONST_PREFIX: &'static str = "DRM_FOURCC_";
 
+    pub fn get_header_include_paths() -> Vec<PathBuf> {
+        let library = pkg_config::Config::new()
+            .probe("libdrm")
+            .expect("Failed to find libdrm");
+
+        library.include_paths
+    }
+
     pub fn generate() -> Result<(), Box<dyn Error + Sync + Send>> {
+        let extra_includes = get_header_include_paths();
         let out_dir = env::var("OUT_DIR").unwrap();
         let wrapper_path = Path::new(&out_dir).join("wrapper.h");
 
         // First get all the macros in drm_fourcc.h
+        let mut cmd = Command::new("clang");
+        cmd.arg("-E") // run pre-processor only
+            .arg("-dM"); // output all macros defined
+
+        for include_path in &extra_includes {
+            cmd.arg("-I").arg(include_path.as_os_str());
```

Comment:
`PathBuf` implements `AsRef<OsStr>`, so you don't even need the `.as_os_str()` here.